gulpfile.js

```diff
@@ -92,7 +107,7 @@ var frontendDependencies = manifest.dependencies.frontend;
 var cssFilesPath = [];
 var jsFilesPath = [];
 var fontFolderPath = [];
-var cssBackgroundPath = [];
+var cssBackgroundFilepaths = [];
 var generatedTargetDir = path.join(
   'third_party', 'generated',
   isMinificationNeeded ? 'prod' : 'dev');
```

Comment:
ooowwh! Now I get the point, we can create the 3 different function that will be called only when inside tasks. Or we can create all three for loops inside the task respectively.
What are your thoughts?